.claude/skills/create-pr/SKILL.md

```
@@ -0,0 +1,32 @@
+1. **ブランチ確認**: main ブランチにいる場合は、新しく feature ブランチを作る
+2. **要件定義署の確認**：/docs/requirements/ 以下に要件定義署が存在する機能の場合、内容を確認してラベルや参照PRの明記などの指示に従う。
+3. **コミット・プッシュ**：コミットされていない変更がある場合、ユーザーに変更をとった上でコミットする。全てのコミットが完了したら、再度ユーザーに確認しプッシュする。
+4. **PR 作成**: 以下のフォーマットでPRを作成する。
+
+### PR フォーマット
+
+- **タイトル**: 変更を一言で表す日本語のタイトル（70文字以内）
+- **assignee**: `SuzukaHori`
+- **本文**:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)